docs/manual/outbackcdx.rst

```diff
@@ -0,0 +1,53 @@
+.. _using-outback:
+
+
+Using OutbackCDX with pywb
+==========================
+
+The recommended setup is to run `OutbackCDX <https://github.com/nla/outbackcdx>`_ alongside pywb.
+OutbackCDX provides an index (CDX) server and can efficient store and lookup web archive data by URL.
+
+
+Adding CDX to OutbackCDX
+------------------------
+
+To setup OutbackCDX, please follow the instructions on the `OutbackCDX README <https://github.com/nla/outbackcdx>`_
+
+Since pywb also uses the default port 8080, be sure to use a different port for OutbackCDX, eg. ``java -jar outbackcdx*.jar -p 8084``.
+
+To add indices to OutbackCDX, index the WARCs using ``cdx-indexer`` command-line tool and then POST to OutbackCDX.
+
+For example, assuming OutbackCDX is running on port 8084:
+
+.. code:: console
+
+    cdx-indexer /path/to/mywarcs/my.warc.gz > ./index.cdx
+    curl -X POST --data-binary @index.cdx http://localhost:8084/mycoll
+
+The contents of the index are added to the ``mycoll`` index, which can correspond to the web archive collection ``mycoll``.
+The index is created automatically if it does not exist.
+
+The above can be repeated for each WARC file, or for the entire collection if small (by using a wildcard, eg. ``cdx-indexer /path/to/mywarcs/*.warc.gz > ./index.cdx``). If the CDX index is too big, OutbackCDX may fail and ingesting an index per-WARC is recommended.
+
```

Comment:
Based on the OutbackCDX docs it seems like if you have cdx files you were already using with OpenWayback, that data can be POSTed to OutbackCDX without reindexing with the pywb `cdx-indexer`. Is that an accurate or is it recommended to reindex the [W]ARCs anyway?

Reply:
There's no need to reindex for the same level of functionality. The only reasons I can think of to reindex are:
- you have old CDX9 indexes without the compressed length field and access the WARC files over HTTP (the length field enables a correct range request for a single warc record)
- you want to index POST requests (I've never tested this myself but there's some discussion about it in #585)